apps/docs/src/examples/extras/trackball-controls/Scene.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import { T } from '@threlte/core'
+  import { TrackballControls } from '@threlte/extras'
+  import { BoxGeometry, MeshStandardMaterial } from 'three'
+</script>
+
+<T.PerspectiveCamera
+  makeDefault
+  position={[10, 5, 10]}
+  lookAt.y={0.5}
+>
+  <TrackballControls />
```

Comment:
Maybe we can also try a `svelte-tweakpane-ui` here to allow users the explore the settings of `<TrackballControls>`? Of course, a similar one should be added to `<OrbitControls>`.

Its likely the file will grow if you do this. In that event, maybe you can pull the `<T.PerspectiveCamera>` block into a separate `Camera.svelte` file to keep it clean.

I guess the tweakpane itself goes in the root `App.svelte` but see how we do it in other examples like `<Gizmo>`.

Reply:
sure, done!

I ended up pulling the camera out of `Scene.svelte` and putting it all in `App.svelte`. It's a lot of properties to pass around, but I could extract `Camera.svelte` if you feel strongly on this!

Reply:
Awesome work! I really like the feel of it.

Some notes:
- [ ] TailwindCSS shouldn't be used in examples.
- [ ] While I do see why you went with putting the camera directly in `App.svelte`, is contradicts our [officially recommended app structure](https://threlte-mytcm6ro7-threlte.vercel.app/docs/learn/basics/app-structure#recommended-app-structure) so I think its better to keep things consistent for pedagogic purposes.
- [ ] The tweakpane consumes a lot of lines and is of little interest pedagogically speaking, so lets move it to its own `Settings.svelte` or `Pane.svelte` component. When imported to `App.svelte`, put it under the `<Canvas>` in the template.
- [ ] Considering the limited screen real-estate, can you experiment and see if reducing the tweakpane's width is feasible?

Reply:
Cool, I nuked all the TailwindCSS in OrbitControls, TrackballControls, TransformControls, and Gizmo. But there's lots of other occurrences in the docs which I'd prefer to change in a separate PR.

I was able to conform to the recommended app structure by splitting out `Settings.svelte` and including it directly in `Scene.svelte`. Does this new arrangement look good to you?

Reducing the tweakpane's width is straightforward:
```
<Pane
  width={150}
  ...
>
```
However, the automatically computed width seems to be the least wide that the TweakPane can be before it starts crunching the contents in a way that clips the edges, so I'm hesitant to introduce that change. Also, if it's a problem the user can collapse the whole TweakPane by clicking on the title of the Pane. Is that sufficient control for it to be a non-issue? Going deeper, the width of this Pane is dictated by how wide the Slider needs to be, but I could not find a straightforward way to control the width of a particular Slider. There is also a `scale` parameter we can play with, but it quickly makes the text hard to read on a non-retina screen. If width is an issue, can you include a screenshot? Maybe I'm just seeing something different than you are!

Reply:
One more request: can you double check the imports in all the files?
- Imports should be before any variables or props.
- Sometimes there are multiple imports from the same package in different lines - they need to be unified to one line.
- Import grouping: Threlte imports should come first, then Three imports and other 3rd party libs, then Svelte imports, and finally local imports. Groups should be separated with a blank line.

Sorry for this request since it should be something automated, but I haven't gotten around to investigate import ordering tools for Svelte.

Reply:
Sounds good! I moved to `<style>` tags, extracted `Settings.svelte` from `Scene.svelte` and put it in `App.svelte`, and fixed all the imports. Ready for another look!